Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -128,15 +129,30 @@ namespace AZ::SceneAPI::Behaviors
         ManifestAction action,
         [[maybe_unused]] RequestingApplication requester)
     {
-        // this toggle makes constructing default mesh groups and a prefab optional
+        using namespace SceneAPI::Events;
+
         if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry)
         {
+            // this toggle makes constructing default mesh groups and a prefab optional
             bool createDefaultPrefab = true;
             settingsRegistry->Get(createDefaultPrefab, s_PrefabGroupBehaviorCreateDefaultKey);
             if (createDefaultPrefab == false)
             {
                 return Events::ProcessingResult::Ignored;
             }
+
+            // do not make a Prefab Group if the animation policy will be applied (if ignore actors is set)
+            bool ignoreActors = true;
+            settingsRegistry->Get(ignoreActors, s_PrefabGroupBehaviorIgnoreActorsKey);
+            if (ignoreActors)
+            {
+                AZStd::set<AZStd::string> appliedPolicySet;
+                GraphMetaInfoBus::Broadcast(&GraphMetaInfoBus::Events::GetAppliedPolicyNames, scene, appliedPolicySet);
```

Comment:
suggestion/question: If we returned the set instead of using an out param then we could call `BroadcastResult` which might make the callsite a bit clearer?